connector_prestashop/tests/test_export_stock_qty_job.py

```diff
@@ -17,9 +17,9 @@ class TestExportStockQuantity(ExportStockQuantityCase):
     def test_job_export_qty(self):
         """ Export a qty on PrestaShop """
         variant_binding = self._create_product_binding(
-            name='Faded Short Sleeves T-shirt',
-            template_ps_id=1,
-            variant_ps_id=1,
+            name='Blouse',
+            template_ps_id=2,
+            variant_ps_id=7,
```

Comment:
```
FAIL: test_import_inventory_record_template (odoo.addons.connector_prestashop.tests.test_import_inventory.TestImportInventory)
` Import the inventory for a template
Traceback (most recent call last):
`   File "connector_prestashop/tests/common.py", line 71, in _decorated
`     result = func(self, *args, **kwargs)
`   File "connector_prestashop/tests/test_import_inventory.py", line 92, in test_import_inventory_record_template
`     self.assertEqual(1799, template.qty_available)
` AssertionError: 1799 != 0.0
```
`test_job_export_qty` executes first and `test_import_inventory_record_template` later. They both use PS product 1. The export sets quantity to 0 and the import fails to assert expected quantity. We have modified `test_job_export_qty` to use PS product 2 and avoid this.